Multidex related crash on Kitkat

Directly after start:
````
E/AndroidRuntime: FATAL EXCEPTION: main
				 Process: org.openhab.habdroid, PID: 3111
				 java.lang.RuntimeException: Unable to get provider com.crashlytics.android.CrashlyticsInitProvider: java.lang.ClassNotFoundException: Didn't find class "com.crashlytics.android.CrashlyticsInitProvider" on path: DexPathList[[zip file "/data/app/org.openhab.habdroid-1.apk"],nativeLibraryDirectories=[/data/app-lib/org.openhab.habdroid-1, /vendor/lib, /system/lib]]
					 at android.app.ActivityThread.installProvider(ActivityThread.java:4793)
					 at android.app.ActivityThread.installContentProviders(ActivityThread.java:4385)
					 at android.app.ActivityThread.handleBindApplication(ActivityThread.java:4325)
					 at android.app.ActivityThread.access$1500(ActivityThread.java:135)
					 at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1256)
					 at android.os.Handler.dispatchMessage(Handler.java:102)
					 at android.os.Looper.loop(Looper.java:136)
					 at android.app.ActivityThread.main(ActivityThread.java:5017)
					 at java.lang.reflect.Method.invokeNative(Native Method)
					 at java.lang.reflect.Method.invoke(Method.java:515)
					 at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:779)
					 at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:595)
					 at dalvik.system.NativeStart.main(Native Method)
				  Caused by: java.lang.ClassNotFoundException: Didn't find class "com.crashlytics.android.CrashlyticsInitProvider" on path: DexPathList[[zip file "/data/app/org.openhab.habdroid-1.apk"],nativeLibraryDirectories=[/data/app-lib/org.openhab.habdroid-1, /vendor/lib, /system/lib]]
					 at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
					 at java.lang.ClassLoader.loadClass(ClassLoader.java:497)
					 at java.lang.ClassLoader.loadClass(ClassLoader.java:457)
					 at android.app.ActivityThread.installProvider(ActivityThread.java:4778)
					 at android.app.ActivityThread.installContentProviders(ActivityThread.java:4385) 
					 at android.app.ActivityThread.handleBindApplication(ActivityThread.java:4325) 
					 at android.app.ActivityThread.access$1500(ActivityThread.java:135) 
					 at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1256) 
					 at android.os.Handler.dispatchMessage(Handler.java:102) 
					 at android.os.Looper.loop(Looper.java:136) 
					 at android.app.ActivityThread.main(ActivityThread.java:5017) 
					 at java.lang.reflect.Method.invokeNative(Native Method) 
					 at java.lang.reflect.Method.invoke(Method.java:515) 
					 at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:779) 
					 at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:595) 
					 at dalvik.system.NativeStart.main(Native Method)
````

--- a/mobile/build.gradle
+++ b/mobile/build.gradle
@@ -13,7 +13,7 @@ android {
         applicationId "org.openhab.habdroid"
         minSdkVersion 14
         targetSdkVersion 25
-        multiDexEnabled = true
+        multiDexEnabled true
         testInstrumentationRunner 'android.support.test.runner.AndroidJUnitRunner'
     }
     buildTypes {

--- a/mobile/src/main/AndroidManifest.xml
+++ b/mobile/src/main/AndroidManifest.xml
@@ -29,6 +29,7 @@
         android:required="false" />
 
     <application
+        android:name="android.support.multidex.MultiDexApplication"
         android:allowBackup="true"
         android:fullBackupContent="true"
         android:icon="@mipmap/icon"

